src/main/resources/sample/failing/TECHNICAL/doesnt-pass-regex-validation.json

```
@@ -0,0 +1,12 @@
+{
+  "requiredStringField": "requiredStringValue",
+  "requiredArray": [
+    "requiredArrayValue1",
+    "requiredArrayValue2",
+    "requiredArrayValue3"
```

```suggestion
    "requiredArrayValue1"
```
Pour limiter la taille des messages ? Et est-ce qu'on ferait pas aussi des périmètres sur ce modèles pour avoir un fichier avec tout et un technical avec que les champs facultatifs pour avoir des messages de tests vraiment réduits ?

---

Au final ces tests ont été migrés sur technical_noreq, sans requiredArray